static/css/timetravel.css

```diff
@@ -44,10 +83,70 @@ html {
 .history__controller > button:hover {
   border-color: rgba(255, 255, 255, 0.1);
   box-shadow: 2px 3px #000;
-  transform: translateY(-1px);
 }
 
 .history__controller > button:active {
   box-shadow: none;
-  transform: translateY(0);
+}
+
+.history__exit {
+  position: absolute;
+  font-size: 1rem;
+  top: 1rem;
+  right: 1rem;
+  padding: 0.2rem;
+}
+
+.pr__title-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.pr__title {
+  font-size: 1.5rem;
+}
+
+.pr__people-wrapper {
+  position: relative;
+  padding: 0.5rem 0;
+}
+
+.pr__people {
+  position: relative;
+  display: block;
+  margin-bottom: 0.5rem;
+}
+
+.pr__avatar {
+  position: relative;
+  width: 2rem;
+  height: 2rem;
+  border-radius: 0.25rem;
+  border: 1px solid rgba(255, 255, 255, 0.2);
+  overflow: hidden;
+}
+
+.pr__people > * {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.pr__mergedAt {
+  font-size: 0.9rem;
+  opacity: 0.5;
+}
+
+@media screen and (min-width: 767px) {
```

Comment:
I'd keep `min-width: 60rem` because this scales up/down if a browser is using a zoom in or out.
And if combined with the `font-size` change from above, breaks correctly when the buttons get too "compressed" :)